lib/unifex/code_generator/base_types/default.ex

```diff
@@ -35,6 +35,15 @@ defmodule Unifex.CodeGenerator.BaseTypes.Default do
     end
 
     @impl true
+    def generate_arg_parse(argument, variable, %{type: :int64} = ctx) do
+      ~g<({
+        ErlNifSInt64 temp = 0;
+        int success = enif_get_#{ctx.type}(env, #{argument}, &temp);
+        #{variable} = (int64_t)temp;
+        success;
+        })>
+    end
+
```

Comment:
This should be in `CodeGenerator.BaseTypes` module responsible for type `:int64`, not here